nemo/collections/asr/data/audio_to_text.py

```diff
@@ -171,47 +171,45 @@ def process_text_by_sample(self, sample: collections.ASRAudioText.OUTPUT_TYPE) -
         return t, tl
 
 
-def expand_audio_filepaths(audio_tar_filepaths, shard_strategy: str, world_size: int, global_rank: int):
+def expand_sharded_filepaths(sharded_filepaths, shard_strategy: str, world_size: int, global_rank: int):
```

Comment:
This one is called in a lot of places so be careful

Reply:
`find . -exec grep expand_audio_filepath {} \; 2>/dev/null` does not find any anymore..